src/components/dialogs/payment-complete.vue

```diff
@@ -35,7 +35,6 @@
 import { Component, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 import PaymentConfirm from '@/components/payment/payment-confirm.vue'
-import RequestDetails from '@/components/common/request-details.vue'
```

Comment:
RequestDetails was not used here.

While in this component, I updated the element name.